lib/internal/modules/run_main.js

```diff
@@ -43,10 +43,29 @@ function runMainESM(mainPath) {
   esmLoader.loadESM((ESMLoader) => {
```

Comment:
the loadESM function here is already supposed to upgrade rejections to uncaught exceptions. is that not working?

Reply:
Hm … for some reason I thought I saw an `UnhandledRejectionWarning` from TLA when trying it out, but you should be right, yes. Not sure what’s up with that, but I’ve updated it to keep using that :+1: